From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 11 Oct 2011 08:47:32 -0600
Patch-mainline: v3.2-rc1
Subject: [PATCH] hv: remove a bunch of unused debug macros from hyperv.h
Git-commit: a832a1eba94096513bc42c96f33957cc46c5f2bf

These aren't used by anyone anymore, so remove them before someone tries
to use them again.

Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/linux/hyperv.h |   24 ------------------------
 1 files changed, 0 insertions(+), 24 deletions(-)

diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index 98a57a5..2229073 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -721,11 +721,8 @@ extern void vmbus_ontimer(unsigned long data);
 							BLKVSC_DRV)
 
 /* Logging Level */
-#define ERROR_LVL				3
 #define WARNING_LVL				4
 #define INFO_LVL				6
-#define DEBUG_LVL				7
-#define DEBUG_LVL_ENTEREXIT			8
 #define DEBUG_RING_LVL				9
 
 extern unsigned int vmbus_loglevel;
@@ -736,33 +733,12 @@ extern unsigned int vmbus_loglevel;
 		printk(KERN_DEBUG #mod": %s() " fmt "\n", __func__, ## args);\
 	} while (0)
 
-#define DPRINT_DBG(mod, fmt, args...) do {\
-	if ((mod & (HIWORD(vmbus_loglevel))) &&		\
-	    (DEBUG_LVL <= LOWORD(vmbus_loglevel)))	\
-		printk(KERN_DEBUG #mod": %s() " fmt "\n", __func__, ## args);\
-	} while (0)
-
-#define DPRINT_INFO(mod, fmt, args...) do {\
-	if ((mod & (HIWORD(vmbus_loglevel))) &&		\
-	    (INFO_LVL <= LOWORD(vmbus_loglevel)))	\
-		printk(KERN_INFO #mod": " fmt "\n", ## args);\
-	} while (0)
-
 #define DPRINT_WARN(mod, fmt, args...) do {\
 	if ((mod & (HIWORD(vmbus_loglevel))) &&		\
 	    (WARNING_LVL <= LOWORD(vmbus_loglevel)))	\
 		printk(KERN_WARNING #mod": WARNING! " fmt "\n", ## args);\
 	} while (0)
 
-#define DPRINT_ERR(mod, fmt, args...) do {\
-	if ((mod & (HIWORD(vmbus_loglevel))) &&		\
-	    (ERROR_LVL <= LOWORD(vmbus_loglevel)))	\
-		printk(KERN_ERR #mod": %s() ERROR!! " fmt "\n",	\
-		       __func__, ## args);\
-	} while (0)
-
-
-
 struct hv_driver;
 struct hv_device;
 
-- 
1.6.0.2


